SofaKernel/framework/sofa/helper/logging/Messaging.h

```diff
@@ -130,7 +130,7 @@
 
 #define FILEINFO(filename, line) sofa::helper::logging::FileInfo(filename, line)
 
-#define FILEINFO(filename, line) sofa::helper::logging::FileInfo(filename, line)
+#define FILEINFO(filename, line) sofa::helper::logging::FileInfo(filename, line)  //rewritten again. delete this?
```

Comment:
Probably, 
it is surprising that the compiler does not complains about redefining something.